aflr/helper_classes.py

```diff
@@ -60,3 +63,27 @@ def download(cls, scriptId, section=None, parameters=None, destination="."):
             local_filenames.append(local_filename)
 
         return local_filenames
+
+
+class UploadableResource(APIRequest):
+    @classmethod
+    def upload_audiofile(cls, file_path):
+        # get presigned URL
+        payload = open(file_path, "rb")
+        filename = os.path.basename(file_path)
+        headers = {"Content-Type": "audio/mpeg"}
```

Comment:
does this not need apikey? why are we not using `headers = cls._build_header()` on ` _get_request()` ?

Reply:
in my point of view _get_request is a method of api_requests which means the helper class inherits it right?
So there is no need for me to call cls._build_header() myself since its in the _get_request()  method.
The header does not have to be added to the get call. Remember ULVC frontend adding the Content-Type in the put request with the presigned url. Reason for this is a wildcard for the Content-Type header in the s3 bucket.